ush/python/pygfs/task/marine_analysis.py

```diff
@@ -284,7 +284,7 @@ def _prep_checkpoint(self: Task) -> None:
 
         # render the SOCA to CICE YAML file for the Arctic and Antarctic
         logger.info("render the SOCA to CICE YAML file for the Arctic and Antarctic")
-        varchgyamls = ['soca_2cice_arctic.yaml', 'soca_2cice_antarctic.yaml']
+        varchgyamls = ['soca_2cice_global.yaml']
```

Comment:
we could probably just get rid of the list and loop, unless we think we might be running more variable changes?